packages/@lwc/integration-karma/test/light-dom/slot-fowarding/slots/reactivity/index.spec.js

```diff
@@ -92,32 +92,48 @@ describe('light DOM slot forwarding reactivity', () => {
         },
     ];
 
-    const expectedSlotContentAfterConditionalMutation = [
-        {
-            slotAssignment: 'lower',
-            slotContent: 'Upper slot content',
-        },
-        {
-            slotAssignment: '',
-            slotContent: 'Default slot content',
-        },
-        {
-            slotAssignment: 'upper',
-            slotContent: 'Lower slot content',
-        },
-        {
-            slotAssignment: 'upper',
-            slotContent: 'Conditional slot content',
-        },
-    ];
-
-    const testCases = ['lightLight', 'lightShadow'];
+    const expectedSlotContentAfterConditionalMutation =
+        process.env.API_VERSION > 60
+            ? [
+                  {
+                      slotAssignment: 'lower',
+                      slotContent: 'Upper slot content',
+                  },
+                  {
+                      slotAssignment: '',
+                      slotContent: 'Default slot content',
+                  },
+                  {
+                      slotAssignment: 'upper',
+                      slotContent: 'Lower slot content',
+                  },
+                  {
+                      slotAssignment: 'upper',
+                      slotContent: 'Conditional slot content',
+                  },
+              ]
+            : [
+                  expectedSlotContentAfterParentMutation[0],
+                  {
+                      slotAssignment: 'upper',
+                      slotContent: 'Conditional slot content',
+                  },
+                  ...expectedSlotContentAfterParentMutation.slice(1),
+              ];
+
+    let testCases = ['lightLight', 'lightShadow'];
 
     if (process.env.NATIVE_SHADOW) {
         // TODO [#3885]: Using expressions on synthetic shadow DOM slots throws an error, only test in native for now
         testCases.push('shadowLight');
     }
 
+    if (process.env.API_VERSION < 61) {
+        // Note for api versions < 61 light DOM slot forwarding is disabled.
+        // Testing with just lightLight should be sufficient as a regression test.
+        testCases = testCases.filter((test) => test === 'lightLight');
+    }
```

Comment:
I'm only testing the light > light scenario as a regression test for the reactivity, since we're preventing the change in light DOM only I don't think we need to test all the scenarios again.

Reply:
The behavior of `lightShadow` is the same in both API versions, correct? If that's the case, then running the tests in all API versions at least boosts our test coverage, and confirms that it didn't change between versions.

Reply:
Updated in 1f9afdcb8f7785b371b53254eb584e9b192ae299!